roles/jumbotron/files/fakta-bell.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+for i in ${1..3}
```

Comment:
This results in "Bad substitution". I'm not sure you even tried running this..

But even with that fixed it doesn't seem to do anything, but that might be just a loose connection. The jumbotron_ping script doesn't do anything either.